segments/path.bash

```diff
@@ -2,6 +2,7 @@
 _sbp_path_color_bg=${_sbp_path_color_bg-:$_sbp_color_blue}
 _sbp_path_color_fg=${_sbp_path_color_fg-:$_sbp_color_white}
 _sbp_path_color_sep=${_sbp_path_color_sep-:$_sbp_color_grey}
+_sbp_path_disable_sep=${_sbp_path_disable_sep-:0}
```

Comment:
I think this should probably be like this:

```bash
_sbp_path_disable_sep="${_sbp_path_disable_sep:-0}"
```

As explained here: http://www.tldp.org/LDP/abs/html/parameter-substitution.html